docs/htdocs/info/docs/tools/vep/vep_faq.html

```diff
@@ -175,9 +175,9 @@ <h2 id="script">Command line VEP questions</h2>
 
 <p>&nbsp;</p>
 
-<p style="font-weight:bold;"> Q: Can I get gnomAD or ExAC allele frequencies in VEP? </p>
+<p style="font-weight:bold;"> Q: Can I get gnomAD exomes and genomes frequencies in VEP? </p>
 
-<p> Yes, see <a href="script/vep_example.html#gnomad">this guide</a>. </p>
+<p> Yes, see <a href="script/vep_example.html#gnomade">this guide</a>. </p>
```

Comment:
This section is still named `gnomad`:
```suggestion
<p> Yes, see <a href="script/vep_example.html#gnomad">this guide</a>. </p>
```